FISH-306 Payara IntelliJ IDEA Plugin documentation (#58)
payara/Payara-Enterprise-Documentation: docs/modules/ROOT/pages/documentation/ecosystem/intellij-plugin/payara-server.adoc

* To deploy the Maven application to Payara Enterprise Server, first create the new *Payara 
Server configuration* by selecting the *Edit Configuration* option:

image::intellij-plugin/payara-server/create-configuration.png[Create Configuration]

* In the *Run/Debug Configurations* dialog, click *+*, expand the *Payara Server* 
node, and select *Local*:

image::intellij-plugin/payara-server/create-local-server-configuration.png[Create Local Server configuration]

* To add the Payara Server to the configuration, click the *Configure...* button 
and browse the Payara Server installation path:

image::intellij-plugin/payara-server/browse-payara-server.png[Browse the Payara Server path]

* Fix the artifacts warnings that appear at the bottom of the run configuration 
settings dialog, by clicking the *Fix* button:

image::intellij-plugin/payara-server/add-artifact.png[Add artifact]

* Select the *sample-micro-app:war* artifact for deployment to the Payara Enterprise Server:

image::intellij-plugin/payara-server/select-war-artifact.png[Select WAR artifact]

* Now fix the debugger related warning by pressing the *Fix* button, which updates 

> the debugger settings in the domains.xml of the selected Payara Enterprise Server�s domain:

Reviewer: ```suggestion
the debugger settings in the domains.xml of the selected Payara Enterprise Server's domain:
```
This is subtle, but the docs will render the existing apostrophe as an unknown character. It's probably not in the correct encoding.

image::intellij-plugin/payara-server/fix-debugger-warning.png[Fix debugger warning]

* Now save the configuration:

image::intellij-plugin/payara-server/save-configuration.png[Save configuration]

* Click the *Run or Debug* button from top-right panel that starts the Payara Server 
and deploys the artifact to the server:

image::intellij-plugin/payara-server/run-server.png[Start the Payara Server]

* After application deployed to the Payara Enterprise Server, IntelliJ starts a web browser 
and opens the application home page URL in the web browser:

image::intellij-plugin/payara-server/home-page.png[Home Page]

> * You may see the log in the *Services ? Payara Log* window:

Reviewer: What's ? in there? Should it be rather an arror, like this:
```suggestion
* You may see the log in the *Services > Payara Log* window:
```

image::intellij-plugin/payara-server/server-log.png[Payara Server log]

> image::intellij-plugin/payara-server/breakpoint-log.png[Debug breakpoint]

Reviewer: The name of the image is just breakpoint.png:
```suggestion
image::intellij-plugin/payara-server/breakpoint.png[Debug breakpoint]
```
